lib/crowbar/init/application.rb

```diff
@@ -230,7 +230,8 @@ def api_constraint(*versions)
             version_mime = %r(^application/vnd\.crowbar\.v(?<major>\d+).(?<minor>\d+)\+json$)
 
             versions_requested = version_mime.match(request.accept.first.entry)
-            return true if versions_requested[:major].to_i == major &&
+            return true if !versions_requested.nil? &&
```

Comment:
actually we don't want to return here, as we have to run into `halt` outside the block. but we can break out of the block